jkq/ddsim/circuitsimulator.py

```diff
@@ -0,0 +1,113 @@
+"""Backend for DDSIM."""
+
+import logging
+import time
+import uuid
+
+from qiskit.providers import BackendV1, Options
+from qiskit import qobj, QiskitError
+from qiskit.providers.models import BackendConfiguration, BackendStatus
+from qiskit.result import Result
+
+from .jkqjob import JKQJob
+from .jkqerror import JKQSimulatorError
+
+from jkq.ddsim import pyddsim
+
+logger = logging.getLogger(__name__)
+
+
+class CircuitSimulator(BackendV1):
+    """Python interface to JKQ DDSIM"""
+
+    @classmethod
+    def _default_options(cls) -> Options:
+        return Options(
+            shots=None,
+            return_statevector=True
+        )
+
+    def __init__(self, configuration=None, provider=None):
+        conf = {
+            'backend_name': 'qasm_simulator',
```

Comment:
Why `qasm_simulator`? wouldn't `circuit_simulator` be more suitable?

Reply:
Good catch!